subsys/infinispan/src/main/resources/infinispan.xml

```diff
@@ -33,34 +25,34 @@
     </local-cache>
 
     <local-cache name="folo-sealed" configuration="local-template">
-      <eviction size="1000" type="COUNT" strategy="LRU"/>
       <persistence passivation="true">
         <file-store shared="false" preload="false" fetch-state="false" path="${indy.data}/folo"/>
       </persistence>
+      <memory>
+        <object size="1000" strategy="LRU" />
+      </memory>
     </local-cache>
 
-    <local-cache name="content-index" configuration="local-template">
-      <eviction strategy="LRU" size="200000" type="COUNT"/>
-    </local-cache>
+    <local-cache name="content-index" configuration="local-template" />
 
     <local-cache name="content-metadata" configuration="local-template"/>
 
     <local-cache name="maven-version-metadata-cache" deadlock-detection-spin="10000" configuration="local-template">
-      <eviction size="10000000" type="COUNT" strategy="LRU"/>
+      <memory>
+        <object size="10000000" strategy="LRU" />
```

Comment:
This needs to be down around a 2000 or so, I'd say, or there will be pretty major disk space / memory problems (depending on whether the cache is persisted or not).